### ISSUE TITLE
Code blocks and Pipe Tables are not rendered correctly

### ISSUE THREAD

**Issue by cnole** [sixro/confluence-maven-plugin#21]
Hey! Great plugin to keep documentation synchronized between Confluence and GitLab. However i have tried it for our project and it turns out that some markdown syntaxes are not rendered correctly, such as code blocks \`\`\`...\`\`\` and tables using pipes. Can you fix it? Thanks!

**Comment by sixro** [comment]
Hi @cnole and thank you.
The project uses a library provided by Atlassian in order to convert Markdown to HTML.
At the moment I have seen that the library is the latest...
The library has been updated by **Philip Strecker** (@puddingspudding) and he added also the library of atlassian handling tables.
Maybe he can explain better the situation.
I'm sorry to not be able to support the project as I wish, but my current company decided not to use this tool... :)
I leave this issue open, hoping that Philip can contribute to the answer...
Anyway thank you for asking the quesiton and your interest in this project :)

**Comment by puddingspudding** [comment]
Hi @cnole,
Are you using the latest version 1.1.0? This version hasn't been uploaded to the maven repository. I don't know if @sixro will upload it since he is not supporting this project anymore. Else you have to install the latest version to you local repository

**Comment by cnole** [comment]
Hi @puddingspudding,

thank you for answering. I am using the version 1.0.3 since it is the latest version in maven repository. I have seen that you added support for pipe tables and hopefully also the code blocks. So the question is are you planning to release the version 1.1.0? @sixro 

That would be very nice.

**Comment by sixro** [comment]
Hi, unfortunately I don't have much time to follow this project...

**Comment by cnole** [comment]
Hi @sixro,

that's OK. I want to ask if i can checkout the source code and publish into our intern Maven Repository. Then i can update or modify the code as i need. Is that OK for you?

**Comment by sixro** [comment]
Of course. If you think there is an issue with the MIT license tell me and
i'll remove it.
And just to share: if you want to create your new plugin starting with this
code, go on. I'll share your project in README...

Il 23/ago/2017 08:48, "Zheng Luo" <notifications@github.com> ha scritto:
